### PR TITLE
feat(tracking): new tracking for solve ctas

### PR DESCRIPTION
**Pull request by runrog**

### CODE REVIEW

**Comment by runrog** [comment]
[run ci]

**Review by WidgetsBurritos — APPROVED** [review]
👍 Approved, but as noted in ticket, we need the class added to the corresponding places in solve/www repos as well.

**Review by WidgetsBurritos — APPROVED** [review]
👍 LGTM

File: styleguide/derek/global-components/tracking/tracking.js [hunk 1]
```diff
@@ -185,6 +185,17 @@ Zoolander.Tracking = (function Tracking($) {
       });
     });
 
+    $(document).on('click', '.solve-cta', (e) => {
+      dataLayer.push({
+        event: 'cta.click',
```
Comment (WidgetsBurritos) [inline styleguide/derek/global-components/tracking/tracking.js]: Just realized, the `cta.click` event is incorrect. Can you change this event to `ga.event` instead?

**Review by WidgetsBurritos — CHANGES_REQUESTED** [review]

**Review by WidgetsBurritos — APPROVED** [review]
👍 on 75ec5b1d5e2f7a3d8ea8dc2218a5c8004698ae07

**Pull request merged**